src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -47,19 +47,19 @@ private bool CheckValidators(ApplicationEngine engine)
         internal override bool Initialize(ApplicationEngine engine)
         {
             if (!base.Initialize(engine)) return false;
-            engine.Snapshot.Storages.Add(CreateStorageKey(Prefix_MaxBlockSize), new StorageItem
+            engine.Snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_MaxBlockSize), () => new StorageItem
```

Comment:
This should be the same, if we already know that the item doesn't exists, why we need to call `GetAndChange`?